.env

```diff
@@ -0,0 +1 @@
+OPENAI_API_KEY=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
```

Comment:
I do not think you want to commit this. You should quickly delete it from your API keys list!!!